fc/jira/issue.py

```diff
@@ -76,6 +76,9 @@ def _extra_json_for_create(self, existing_json: dict):
     def type_str(self) -> str:
         raise NotImplementedError
 
+    def update_vfr(self, duration: int, cost_of_delay: int) -> float:
```

Comment:
Updating the VFR doesn't make sense in for a generic `Issue` that could span across any Jira project.  What if we were to put this in `Task` (soon to be `FcIssue`)?

Reply:
This was moved to be defined in `FcIssue`, but modified as per next comment.